src/app/public/modules/dropdown/dropdown.component.scss

```diff
@@ -24,7 +24,14 @@
 
 .sky-dropdown-button-type-tab {
   @include sky-btn-tab();
-  @include sky-btn-tab-selected();
+
+  // This 'not' psuedo-class isn't totally necessary other than it allows this selector to be
+  // specific enough not to be overwritten. Other styles will overule this style unless the
+  // 'not' psuedo-class is added
+  &:not(.sky-btn-tab-disabled) {
```

Comment:
Which styles from the `sky-btn-tab-selected()` need to be overwritten by tabs? Would it be better to overwrite those in the tabs library, so that we don't have the `.sky-btn-tab-disabled` CSS class name "bleeding" into the popovers library?

Another thought would be to put the `:not` in the `sky-btn-tab-selected()` mixin.

Reply:
So the issue wasn't that any styles from `sky-btn-tab-selected()` needed to be overwritten. The issue was that the selector wasn't specific enough and things in `sky-btn-tab-selected()` were being overwritten by things in `sky-btn-tab()` if they were used together. I think moving the `:not` into the mixin is probably the best move. Going to close this PR and make a new one in `theme` that does that.